hazelcast-jet-core/src/main/java/com/hazelcast/jet/Sinks.java

```diff
@@ -58,6 +63,82 @@ private Sinks() {
         return new SinkImpl<>(sinkName, metaSupplier);
     }
 
+    /**
+     * Returns a sink that updates {@code Map.Entry}s it receives into a Hazelcast
+     * {@code IMap} with the specified name and update function. The function
+     * will be wrapped inside an {@link EntryProcessor}.
```

Comment:
this should include some code similar to `Map.compute`, it's not clear which parameter goes where currently. Also not clear what happens when the key doesn't exist in the map.